src/tools/performance_test/perf_test.py

```diff
@@ -0,0 +1,244 @@
+"""
+This test should be run using python3 from directory containing esc binary (or symbolic link)
+Node 1 must be running
+
+1) create 10k accounts
+2) accounts 0..7 receive 1M esc, other accounts 1 esc
+3) SEND_ONE TEST: accounts 0..7 send 10 esc to accounts 6001..7250 (10k transactions in total)
+4) create token accounts for all users (10k) for token 0
+5) account 0 adds 10M own tokens and moves 1M own tokens to accounts 1..7
+6) MOVE_TOKENS TEST: accounts 0..7 move 10 tokens to accounts 6001..7250 (10k transactions in total)
+"""
+
+from time import *
+from multiprocessing import Process
+import os
+
+
+def get_account_address(account_id):
+    return '0001-' + str(hex(account_id)[2:]).rjust(8, '0') + '-XXXX'
+
+
+def prepare_accounts(num_accounts, num_esc):
+    create_account_directories(num_esc)
+    create_accounts(num_esc, num_accounts)
+
+
+def create_account_directories(num_accounts):
+    for i in range(num_accounts):
+        path = 'user_' + str(i)
+        if not os.path.exists(path):
+            os.makedirs(path)
+            os.chdir(path)
+            os.symlink('../esc', 'esc')
+            with open('settings.cfg', 'w') as settings:
+                settings.write('port=9091\n')
+                settings.write('host=127.0.0.1\n')
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this IP "127.0.0.1" address configurable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS1313)